system/Validation/FileRules.php

```diff
@@ -260,4 +260,51 @@ public function max_dims(?string $blank, string $params): bool
 
         return true;
     }
+
+    /**
+     * Checks an uploaded file to verify that the dimensions are greater than
+     * a specified dimension.
+     */
+    public function min_dims(?string $blank, string $params): bool
+    {
+        // Grab the file name off the top of the $params
+        // after we split it.
+        $params = explode(',', $params);
+        $name   = array_shift($params);
+
+        if (! ($files = $this->request->getFileMultiple($name))) {
```

Comment:
Now we accept only **booleans** for if condition.
Please rewrite the condition.